packages/datatrak-web/src/views/LandingPage.tsx

```diff
@@ -5,13 +5,15 @@
 
 import React from 'react';
 import { ButtonLink, PageContainer } from '../components';
+import { TopProgressBar } from '../components/TopProgressBar';
```

Comment:
Should export `TopProgressBar` from `components/index.ts` so that we can import this like `import { TopProgressBar } from '../components'`